Makefile

```diff
@@ -177,5 +177,4 @@ generate-proto-py:
 
 
 black:
-	black --line-length 100 --target-version py36 --exclude "/(\.eggs|\.git|\.hg|\.mypy_cache|\.nox|\.tox|\.venv|_build|buck-out|build|dist|buildman)/" . # TODO(kleesc): Re-enable after buildman rewrite
-
+	black --line-length=100 --target-version=py38 --check --diff .
```

Comment:
@kurtismullins sorry, only remove the `buildman` from list, not the entire list

Reply:
No problem! I put the exclusions back in.